addons/website/static/tests/tours/link_tools.js

```diff
@@ -285,79 +347,92 @@ registerWebsitePreviewTour('link_tools', {
     },
     {
         content: "Open link tools",
-        trigger: "#toolbar #create-link",
-        run: "click",
+        trigger: ".o-we-toolbar div[name='link'] button",
+        run: 'click',
     },
     clickOnImgStep,
     {
         // URL transformation into link should persist, without the need for
         // input at input[name=url]
         content: "Check that link was created",
-        trigger: ":iframe .s_text_image p a[href='http://odoo.com']:contains('odoo.com')",
+        trigger: ":iframe .s_text_image p a[href='https://odoo.com']:contains('odoo.com')",
     },
+    clickOnLink(":iframe .s_text_image p a[href='https://odoo.com']"),
+    // 8. Check that http links are not coerced to https and vice-versa.
     {
-        content: "Click on link to open the link tools",
-        trigger: ":iframe .s_text_image p a[href='http://odoo.com']",
+        content: "Click on edit button in link popover",
+        trigger: ".o-we-linkpopover .o_we_edit_link",
         run: "click",
     },
-    // 8. Check that http links are not coerced to https and vice-versa.
     {
-        content: "Change URL to https",
-        trigger: "#o_link_dialog_url_input",
-        run() {
-            // TODO: update the tour to use helpers.edit("https://odoo.com")
-            // To see what happens with edit, add `pause:true` to the previous step
-            // and type yourself https://odoo.com in #o_link_dialog_url_input
-            // The label will be ohttps://
-            this.anchor.value = "https://odoo.com";
-            this.anchor.dispatchEvent(new InputEvent("input", { bubbles: true }));
-        }
+        content: "Change content (editing the label input) to odoo website_2",
```

Comment:
```suggestion
        content: "Change the URL back to http",
```
Previously, the default created website used `http`, but now it uses the `https` route. Please verify if this step is necessary.